### PR TITLE
Basic login

### PR DESCRIPTION
**Pull request by nguyenquangdung14cntt**
screenshot from 2017-12-15 15 18 29

### CODE REVIEW

**Comment by nguyenquangdung14cntt** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
+      flash[:success] = t "success_log_in"
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = t "error_log_in"
+      render "new"
```
Comment (tranducquoc) [inline app/controllers/sessions_controller.rb]: render :new

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<% provide :title, t(".title") %>
+<h1>Log in</h1>
```
Comment (tranducquoc) [inline app/views/sessions/new.html.erb]: I18n

**Comment by tranducquoc** [comment]
commented

**Comment by nguyenquangdung14cntt** [comment]
ready

**Comment by VuThiTranVan** [comment]
Merged

**Pull request merged**